### PR TITLE
Typo (add codeface)

### LINKED ISSUE #31858: errata corrige
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_operators

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

x **= f() evaluates into the resulting power x ** y

### What did you expect to see?

x **= f() evaluates into the resulting power x ** f()

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/guide/expressions_and_operators`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_operators
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/guide/expressions_and_operators/index.md
* Last commit: https://github.com/mdn/content/commit/cf1c22caadad254455435bee86433afa9e9470ce
* Document last modified: 2023-11-23T17:41:10.000Z

</details>

### PR DESCRIPTION
**Pull request by dawei-wang**
Fix mdn/content#31858

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
👍

File: files/en-us/web/javascript/guide/expressions_and_operators/index.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ console.log(f(0, x = f(), 0));
 The evaluation result matches the expression to the right of the `=` sign in the
 "Meaning" column of the table above. That means that `x = f()` evaluates into
 whatever `f()`'s result is, `x += f()` evaluates into the resulting sum `x + f()`,
-`x **= f()` evaluates into the resulting power `x ** y`, and so on.
+`x **= f() evaluates into the resulting power x ** f(), and so on.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/expressions_and_operators/index.md:187]: @dawei-wang Why did you remove the backticks?

Comment (teoli2003) [reply files/en-us/web/javascript/guide/expressions_and_operators/index.md:187]: I misread.

I read in the opposite direction.

Reverting
